server/routes/content.js

```diff
@@ -23,9 +23,14 @@ const createContentRouter = ({ hubContentService, analyticsService }) => {
 
     const establishmentId = path(['session', 'establishmentId'], req);
     const userAgent = path(['headers', 'user-agent'], req);
+    const { establishmentName } = req.session;
 
     try {
-      const data = await hubContentService.contentFor(id, establishmentId);
+      const data = await hubContentService.contentFor(
+        id,
+        establishmentId,
+        establishmentName,
```

Comment:
we'll eventually swap this out and just call the cms service and only pass establishment name